.github/workflows/nucliadb_one.yml

```diff
@@ -130,13 +130,17 @@ jobs:
       - name: Build local ClusterManager for testing
         uses: docker/build-push-action@v4
         if: steps.filter.outputs.cluster_modified == 'true'
-        with:
-          context: .
-          file: Dockerfile.cluster_monitor
-          push: false
-          cache-from: type=gha
-          cache-to: type=gha,mode=min
-          tags: eu.gcr.io/stashify-218417/cluster_manager:main
+        run: |
+          docker build -t eu.gcr.io/stashify-218417/cluster_manager:main -f Dockerfile.cluster_monitor .
+
+        # TODO: un-comment when we sort out image cache issues
```

Comment:
Just remove this now. We don't need the commented code as we have other examples on how to set it up. Also can you fix the 2 above this?